prow/cmd/deck/static/prow/prow.ts

```diff
@@ -739,15 +725,41 @@ function redraw(fz: FuzzySearch, pushState: boolean = true): void {
 function createRerunCell(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string): HTMLTableDataCellElement {
     const c = document.createElement("td");
     c.appendChild(createRerunProwJobIcon(modal, rerunElement, prowjob, rerunCreatesJob, csrfToken));
-    c.classList.add("icon-cell");
     return c;
 }
 
+function createLogCell(build: ProwJob, buildUrl: string): HTMLTableDataCellElement {
+    const { agent, job, pod_spec } = build.spec
```

Comment:
I just _adore_ destructuring assignments. 💯 

It _is_ unfortunate that the properties are "snake_case" (and not `correctlyCamelCased` 🤣 ).

Reply:
I agree :( However, fixing this may be out of scope for this PR